helm/alfresco-identity-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: alfresco-identity-service
-version: 7.0.0
-appVersion: 1.8.0
+version: 8.0.0
```

Comment:
I would not bump the chart major version if we are releasing just an hotfix, `7.0.1` should be fine

Reply:
Changed version from `8.0.0` to `7.0.1`